public/app/plugins/datasource/loki/components/LokiContextUi.test.tsx

```diff
@@ -136,8 +136,8 @@ describe('LokiContextUi', () => {
     await waitFor(() => {
       expect(props.logContextProvider.getInitContextFilters).toHaveBeenCalled();
     });
-    const select = await screen.findAllByRole('combobox');
-    await selectOptionInTest(select[0], 'label1="value1"');
+    const selects = await screen.findAllByRole('combobox');
+    await waitFor(() => select(selects[0], 'label1="value1"', { container: document.body }));
```

Comment:
I'm seeing that you can await for select, without having to use `waitFor`. https://testing-library.com/docs/ecosystem-react-select-event/

Reply:
Updated in https://github.com/grafana/grafana/pull/80546/commits/a4e355685ac89717ded1960d84e4aedee2f21b0e